openwisp_notifications/tests/test_notifications.py

```diff
@@ -944,6 +944,61 @@ def test_notification_for_unverified_email(self):
         # we don't send emails to unverified email addresses
         self.assertEqual(len(mail.outbox), 0)
 
+    @patch('openwisp_notifications.tasks.send_batched_email_notifications.apply_async')
+    def test_batch_email_notification(self, mock_send_email):
```

Comment:
@Dhanus3133 none of these test verifies the actual formatting of the email. We do not have tests for the text email at all.

Reply:
Updated!